src/main/java/me/apisek12/StoneDrop/InventorySelectors/InventorySelector.java

```diff
@@ -0,0 +1,215 @@
+    protected Player player;
+    protected LinkedHashMap<String, Setting> settings;
+    protected static String title = ChatColor.DARK_AQUA + Message.GUI_TITLE.toString();
+    protected Inventory selector;
+    protected static HashMap<Player, InventorySelector> objects = new HashMap<>();
+    protected boolean willBeUsed = false;
+    protected static ItemStack exit, back;
+    protected ItemStack cobble;
+    protected ArrayList<ItemStack> items = new ArrayList<>();
+    static {
+        exit = new ItemStack(Material.BARRIER);
+        ItemMeta exitMeta = exit.getItemMeta();
+
+        exitMeta.setDisplayName(ChatColor.RED+Message.GUI_EXIT_BUTTON.toString());
+        exitMeta.addItemFlags(ItemFlag.HIDE_ATTRIBUTES);
+
+        exit.setItemMeta(exitMeta);
+    }
+
+    public InventorySelector() {}
+
+    public InventorySelector(Player player, LinkedHashMap<String, Setting> settings) {
+        this.player = player;
+        this.settings = new LinkedHashMap<>();
+        this.settings.putAll(settings);
+        objects.put(player, this);
+        selector = Bukkit.createInventory(null, PluginMain.dropChances.size() + (9 - PluginMain.dropChances.size() % 9) + 2 * 9, title);
+        
+        this.cobble = new ItemStack(Material.COBBLESTONE);
+
+        this.refreshCobbleObject();
+        reloadInventory();
+        if (PluginMain.plugin.versionCompatible(12)) player.playSound(player.getLocation(), Sound.ENTITY_ENDERMAN_TELEPORT, (float)PluginMain.volume, 0);
+        player.openInventory(selector);
+    }
+
+    protected String setLoreLine(String chance, String amount,String enchant){
+       return String.format("%s%-14s%s%10s%10s",ChatColor.GOLD,enchant,ChatColor.GRAY,chance,amount);
+    }
+
+    protected ArrayList<String> setDropItemLore(DropChance dropData, Setting setting){
+        ArrayList<String> lore = new ArrayList<>();
+        String onOff;
+        if (setting.isOn()) onOff = ChatColor.GREEN + Message.INFO_ENABLED.toString();
+        else onOff = ChatColor.RED + Message.INFO_DISABLED.toString();
+        lore.add(ChatColor.GRAY+ Message.GUI_ITEM_LEVEL_IN_RANGE.toString()+": "+ChatColor.GOLD+dropData.getMinLevel()+"-"+dropData.getMaxLevel());
+        lore.add(ChatColor.GRAY + Message.GUI_ITEM_DESCRIPTION_THIS_ITEM_DROP_IS.toString()+ " " + onOff + ".");
+        lore.add("");
+        DecimalFormat format = new DecimalFormat("##0.0##");
+        for(int f_num=0;f_num<=3;f_num++){
+            double chance = dropData.getFortuneChance(f_num);
+            if(chance>0){
+                String amountStr = String.valueOf((int)dropData.getFortuneItemsAmountMin(f_num))+ " - "+
+                        String.valueOf((int)dropData.getFortuneItemsAmountMax(f_num));
+                lore.add(setLoreLine(
+                        format.format(chance*100)+"%",
+                        amountStr,
+                        "fortune "+f_num+":"
+                        )
+                );
+
+            }
```

Comment:
To rozwiązanie zmniejsza możliwości personalizacji (zmiana języka lang.yml).
Chyba najlepiej byłoby jednak zamiast loopować, to dodać osobno dla każdej fortuny linię (w ten sposób będziemy mogli do Messages dodać odpowiednią wiadomość, aby admini mogli zmieniać sobie język, a i tak na razie w mc są tylko 3 fortuny różne).